### PR TITLE
#145 update authorization page

### PR DESCRIPTION
**Pull request by Lvyshnevska**

### CODE REVIEW

**Review by BelousSofiya — APPROVED** [review]

**Review by Shushunya — APPROVED** [review]

File: FrontEnd/src/App.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ function App() {
         <Header isAuthorized={true}></Header>
         <Routes>
           <Route path="/" element={<MainPage />} />
-          <Route path="/authorization" element={<MainPageRectangle />} />
+          <Route path="/authorization" element={<AuthorizationPage />} />
           <Route path="/SignUpPage" element={<SignUpPage />} />
           <Route path="/PrivacyPolicyPage" element={<PrivacyPolicy />} />
```
Comment (popovycholeg) [inline FrontEnd/src/App.js:31]: There is no need to add `Page` to the path. Usually, for URLs, we divide words using a dash. 
Example `privacy-policy`

File: FrontEnd/src/components/authorization/AuthorizationPage.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+import Login from "../authorization/Login";
+import AlertMessage from "../authorization/AlertMessage";
+import classes from "./AuthorizationPage.module.css";
+import dots_decor from "./dots_decor.svg";
+
+const AuthorizationPage = (props) => {
+  const [errorText, setErrorText] = useState("");
+
+
```
Comment (popovycholeg) [inline FrontEnd/src/components/authorization/AuthorizationPage.jsx:10]: Setup linter, two new lines are redundant

File: FrontEnd/src/components/authorization/AuthorizationPage.module.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
+.auth-page {
+    display: flex;
+    width: 1512px;
+    height: 725px;
+    flex-shrink: 0;
+    background: #f1fff7;
+  }
+  
+
```
Comment (popovycholeg) [inline FrontEnd/src/components/authorization/AuthorizationPage.module.css:9]: same as above

File: FrontEnd/src/components/authorization/AuthorizationPage.module.css [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  }
+  
+  .frame-img-right {
+    display: inline-flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 24px;
+    margin: 102px 354px 473px 979px;
+    position: absolute;
+  }
```
Comment (popovycholeg) [inline FrontEnd/src/components/authorization/AuthorizationPage.module.css:26]: new line at the end of the file
https://stackoverflow.com/a/729795/9918279

File: FrontEnd/src/components/authorization/LoginContent.jsx [hunk 5]
```diff
@@ -34,28 +33,20 @@ const LoginContent = (props) => {
...
       });
     } else if (enteredPassword.trim().length === 0) {
       errorMessage = emptyFieldErrorText;
       setError({
-        errorType: "empty-password-field",
+        errorType: "required",
         message: emptyFieldErrorText,
       });
     } else if (
+      // fake validation to show functionality
```
Comment (popovycholeg) [inline FrontEnd/src/components/authorization/LoginContent.jsx:49]: Why do we need fake validation?

File: FrontEnd/src/components/authorization/LoginContent.jsx [hunk 6]
```diff
@@ -111,16 +101,16 @@ const LoginContent = (props) => {
                   value={enteredEmail}
                   onChange={emailChangeHandler}
                 />
-                <span className={classes["error-message"]}>
-                  {error.errorType === "email" && error.message}
-                </span>
               </div>
+              <span className={classes["error-message"]}>
+                {error.errorType === "email" && error.message}
```
Comment (popovycholeg) [inline FrontEnd/src/components/authorization/LoginContent.jsx:106]: consider using [react-hook-form](https://www.npmjs.com/package/react-hook-form) in future to simplify validation work

File: FrontEnd/src/components/authorization/LoginContent.module.css [hunk 7]
```diff
@@ -268,3 +264,24 @@
...
+  letter-spacing: -0.14px;
+}
+
+.forget-password:focus {
+  outline: none;
+}
+
+.password-visibility {
+  cursor: pointer;
+}
```
Comment (popovycholeg) [inline FrontEnd/src/components/authorization/LoginContent.module.css:287]: New line at the end

File: FrontEnd/src/components/authorization/LoginContent.jsx (hunk 5 above)
Comment (Lvyshnevska) [reply FrontEnd/src/components/authorization/LoginContent.jsx:49]: I've added this fake field validation to demonstrate functionality (appropriate error messages) on our previous demo. But this fields should be validated on back-end side. Could you recommend please, how can I handle this validation now on front-end side?

File: FrontEnd/src/components/authorization/LoginContent.jsx (hunk 6 above)
Comment (Lvyshnevska) [reply FrontEnd/src/components/authorization/LoginContent.jsx:106]: Thanks for recommendation. I'll try to use it next time.

File: FrontEnd/src/App.js (hunk 1 above)
Comment (Lvyshnevska) [reply FrontEnd/src/App.js]: Fixed

File: FrontEnd/src/components/authorization/AuthorizationPage.jsx (hunk 2 above)
Comment (Lvyshnevska) [reply FrontEnd/src/components/authorization/AuthorizationPage.jsx]: Fixed

File: FrontEnd/src/components/authorization/AuthorizationPage.module.css (hunk 3 above)
Comment (Lvyshnevska) [reply FrontEnd/src/components/authorization/AuthorizationPage.module.css]: Fixed

File: FrontEnd/src/components/authorization/AuthorizationPage.module.css (hunk 4 above)
Comment (Lvyshnevska) [reply FrontEnd/src/components/authorization/AuthorizationPage.module.css]: Fixed

File: FrontEnd/src/components/authorization/LoginContent.module.css (hunk 7 above)
Comment (Lvyshnevska) [reply FrontEnd/src/components/authorization/LoginContent.module.css]: Fixed

File: FrontEnd/src/components/authorization/LoginContent.jsx [hunk 8]
```diff
@@ -138,35 +128,33 @@ const LoginContent = (props) => {
                     value={enteredPassword}
                     onChange={passwordChangeHandler}
                   />
-                  <button onClick={passwordVisisbilityHandler} type="button">
+                  <span className={classes["password-visibility"]} onClick={passwordVisisbilityHandler}>
```
Comment (MMashivskij) [inline FrontEnd/src/components/authorization/LoginContent.jsx:131]: Why did change `button` to `span`?

Comment (Lvyshnevska) [reply FrontEnd/src/components/authorization/LoginContent.jsx:131]: It's only an icon element for handling password visibility. It's more convenient to style this element inside of input element. And I've investigated such elements on the Internet, most of them were added as span element.

**Pull request merged**